cli/src/test/java/com/box/l10n/mojito/cli/command/ExtractionDiffNotificationCommandTest.java

```diff
@@ -150,18 +150,21 @@ public void sendNotification() throws Exception {
     verify(mockDifferentialRevision, times(1))
         .addComment(
             "{objectId}",
-            "⚠️ 5 strings removed and 2 strings added (from 10 to 7) in diff: ${DIFF_ID}. Check [[https://build.org/${BUILD_NUMBER}|build]] for extraction details.");
+            "⚠️ 5 strings removed [1 hour, 1 month, 1 day, 1 hour, 1 month] and 2 strings added [1 hour update, 1 day update] (from 10 to 7) in diff: ${DIFF_ID}. Check [[https://build.org/${BUILD_NUMBER}|build]] for extraction details.");
```

Comment:
Can we move the list of strings to the end of the message? 

The first sentence should contain the critical information/summary and the link to the build. Then have the 2 lists as they can get pretty long

Reply:
Ok. Could you give a sample of the expected message you would like to see?

Reply:
chatted offline